db/migrate/20160922144222_change_fact_name_to_puppet_fact_name.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeFactNameToPuppetFactName < ActiveRecord::Migration
+  def up
+    FactName.select { |f| f.type == 'FactName' }.each do |fact|
```

Comment:
`FactName.update_all` retrieves `ForemanAnsible::FactName` too, so that can't  be done, but I can chain where and update_all